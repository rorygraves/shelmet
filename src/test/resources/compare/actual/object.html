<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8"/>
    <title>instance of java.net.SocksSocketImpl #10597</title>
    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
    <meta name="description" content=""/>
    <meta name="author" content=""/>

    <link href="/css/bootstrap.css" rel="stylesheet"/>
    <style type="text/css">
      body {
        padding-top: 60px;
        padding-bottom: 40px;
      }
    </style>
    <link href="/css/bootstrap-responsive.css" rel="stylesheet"/>

    <!-- HTML5 shim, for IE6-8 support of HTML5 elements -->
    <!--[if lt IE 9]>
      <script src="/js/html5shiv.js"></script>
    <![endif]-->
  </head>
  <body>
    <div class="navbar navbar-inverse navbar-fixed-top">
      <div class="navbar-inner">
        <div class="container">
          <button type="button" class="btn btn-navbar" data-toggle="collapse" data-target=".nav-collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </button>
          <a class="brand" href="/">SHelmet</a>
          <div class="nav-collapse collapse">
            <ul class="nav">
              <li class="active"><a href="/">Home</a></li>
              <li class="dropdown">
                <a href="#" class="dropdown-toggle" data-toggle="dropdown">Reports<b class="caret"></b></a>
                <ul class="dropdown-menu">
                  <li class="nav-header">Classes</li>
                  <li><a href="/allClassesWithPlatform">Including platform</a></li>
                  <li><a href="/allClassesWithoutPlatform">Excluding platform</a></li>
                  <li class="divider"></li>
                  <li><a href="/rootSet">RootSet</a></li>
                  <li class="divider"></li>
                  <li class="nav-header">Instance Counts</li>
                  <li><a href="/showInstanceCountsIncPlatform">Including platform</a></li>
                  <li><a href="/showInstanceCountsExcPlatform">Excluding platform</a></li>
                  <li class="divider"></li>
                  <li><a href="/histogram">Heap histogram</a></li>
                  <li class="divider"></li>
                  <li><a href="/finalizerSummary">Finalizer summary</a></li>
                </ul>
              </li>
              <li><a href="/about">About</a></li>
            </ul>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container">

<h1>instance of java.net.SocksSocketImpl #10597</h1>
<table class="table table-bordered table-condensed">
<tr>
<td>
<b>Instance of:</b>
</td>
<td>
<a href="/object/0x7fb29c118">class java.net.SocksSocketImpl</a>
</td>
</tr>
<tr>
<td>
<b>Size</b>
</td>
<td>
120 bytes
</td>
</tr>
<tr>
<td>
<b>Retained Size</b>
</td>
<td>
152 bytes
</td>
</tr>
<tr>
<td>
<b>References to this object</b>
</td>
<td>
<a href="#refers">3</a>
</td>
</tr>
<tr>
<td>
<b>Root References</b>
</td>
<td>
None
</td>
</tr>
</table>
<h2>Instance Fields:</h2>
<table class="table table-bordered table-condensed">
<tr>
<th>
Field
</th>
<th>
Type
</th>
<th>
Value
</th>
</tr>
<tr>
<td>
applicationSetProxy
</td>
<td>
boolean
</td>
<td>
false</td>
</tr>
<tr>
<td>
cmdOut
</td>
<td>
ref
</td>
<td>
&lt;null&gt;</td>
</tr>
<tr>
<td>
cmdIn
</td>
<td>
ref
</td>
<td>
&lt;null&gt;</td>
</tr>
<tr>
<td>
cmdsock
</td>
<td>
ref
</td>
<td>
&lt;null&gt;</td>
</tr>
<tr>
<td>
useV4
</td>
<td>
boolean
</td>
<td>
false</td>
</tr>
<tr>
<td>
external_address
</td>
<td>
ref
</td>
<td>
&lt;null&gt;</td>
</tr>
<tr>
<td>
port
</td>
<td>
int
</td>
<td>
1080</td>
</tr>
<tr>
<td>
server
</td>
<td>
ref
</td>
<td>
&lt;null&gt;</td>
</tr>
<tr>
<td>
stream
</td>
<td>
boolean
</td>
<td>
true</td>
</tr>
<tr>
<td>
lastfd
</td>
<td>
int
</td>
<td>
-1</td>
</tr>
<tr>
<td>
anyLocalBoundAddr
</td>
<td>
ref
</td>
<td>
<a href="/object/0x7f4555a30">java.net.Inet4Address @ 0x7f4555a30 (24 bytes)</a>
</td>
</tr>
<tr>
<td>
fd1
</td>
<td>
ref
</td>
<td>
&lt;null&gt;</td>
</tr>
<tr>
<td>
resetLock
</td>
<td>
ref
</td>
<td>
<a href="/object/0x7f4554e48">java.lang.Object @ 0x7f4554e48 (16 bytes)</a>
</td>
</tr>
<tr>
<td>
resetState
</td>
<td>
int
</td>
<td>
0</td>
</tr>
<tr>
<td>
CONNECTION_RESET
</td>
<td>
int
</td>
<td>
2</td>
</tr>
<tr>
<td>
CONNECTION_RESET_PENDING
</td>
<td>
int
</td>
<td>
1</td>
</tr>
<tr>
<td>
CONNECTION_NOT_RESET
</td>
<td>
int
</td>
<td>
0</td>
</tr>
<tr>
<td>
closePending
</td>
<td>
boolean
</td>
<td>
false</td>
</tr>
<tr>
<td>
fdLock
</td>
<td>
ref
</td>
<td>
<a href="/object/0x7f4554e30">java.lang.Object @ 0x7f4554e30 (16 bytes)</a>
</td>
</tr>
<tr>
<td>
fdUseCount
</td>
<td>
int
</td>
<td>
1</td>
</tr>
<tr>
<td>
socketInputStream
</td>
<td>
ref
</td>
<td>
&lt;null&gt;</td>
</tr>
<tr>
<td>
shut_wr
</td>
<td>
boolean
</td>
<td>
false</td>
</tr>
<tr>
<td>
shut_rd
</td>
<td>
boolean
</td>
<td>
false</td>
</tr>
<tr>
<td>
trafficClass
</td>
<td>
int
</td>
<td>
0</td>
</tr>
<tr>
<td>
timeout
</td>
<td>
int
</td>
<td>
0</td>
</tr>
<tr>
<td>
localport
</td>
<td>
int
</td>
<td>
7532</td>
</tr>
<tr>
<td>
port
</td>
<td>
int
</td>
<td>
0</td>
</tr>
<tr>
<td>
address
</td>
<td>
ref
</td>
<td>
<a href="/object/0x7f4555a30">java.net.Inet4Address @ 0x7f4555a30 (24 bytes)</a>
</td>
</tr>
<tr>
<td>
fd
</td>
<td>
ref
</td>
<td>
<a href="/object/0x7f4555a60">java.io.FileDescriptor @ 0x7f4555a60 (24 bytes)</a>
</td>
</tr>
<tr>
<td>
serverSocket
</td>
<td>
ref
</td>
<td>
<a href="/object/0x7f4554a00">java.net.ServerSocket @ 0x7f4554a00 (24 bytes)</a>
</td>
</tr>
<tr>
<td>
socket
</td>
<td>
ref
</td>
<td>
&lt;null&gt;</td>
</tr>
</table>
<h2>References to this object:</h2>
<a name="refers"></a>
<a href="/object/0x7f453a168">java.lang.Thread @ 0x7f453a168  Monitor Ctrl-Break (112 bytes)</a>
 : <br/>
<a href="/object/0x7f4554a00">java.net.ServerSocket @ 0x7f4554a00 (24 bytes)</a>
 : field impl<br/>
<a href="/object/0x7f4554e08">java.lang.ref.Finalizer @ 0x7f4554e08 (40 bytes)</a>
 : field referent<br/>
<h2>Other Queries</h2>
Reference Chains from Rootset
<ul>
<li>
<a href="/objectRootsExcWeak/0x7f4554d90">Exclude weak refs</a>
</li>
<li>
<a href="/objectRootsIncWeak/0x7f4554d90">Include weak refs</a>
</li>
</ul>
<br/>
    </div> <!-- /container -->
    <script src="/js/jquery-2.0.3.min.js"></script>
    <script src="/js/bootstrap.js"></script>
  </body>
</html>

