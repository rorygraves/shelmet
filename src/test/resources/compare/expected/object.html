<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8"/>
    <title>instance of java.net.SocksSocketImpl #10597</title>
    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
    <meta name="description" content=""/>
    <meta name="author" content=""/>

    <link href="/css/bootstrap.css" rel="stylesheet"/>
    <style type="text/css">
      body {
        padding-top: 60px;
        padding-bottom: 40px;
      }
    </style>
    <link href="/css/bootstrap-responsive.css" rel="stylesheet"/>

    <!-- HTML5 shim, for IE6-8 support of HTML5 elements -->
    <!--[if lt IE 9]>
      <script src="/js/html5shiv.js"></script>
    <![endif]-->
  </head>
  <body>
    <div class="navbar navbar-inverse navbar-fixed-top">
      <div class="navbar-inner">
        <div class="container">
          <button type="button" class="btn btn-navbar" data-toggle="collapse" data-target=".nav-collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </button>
          <a class="brand" href="/">SHelmet</a>
          <div class="nav-collapse collapse">
            <ul class="nav">
              <li class="active"><a href="/">Home</a></li>
              <li class="dropdown">
                <a href="#" class="dropdown-toggle" data-toggle="dropdown">Reports<b class="caret"></b></a>
                <ul class="dropdown-menu">
                  <li class="nav-header">Classes</li>
                  <li><a href="/allClassesWithPlatform/">Including platform</a></li>
                  <li><a href="/allClassesWithoutPlatform/">Excluding platform</a></li>
                  <li class="divider"></li>
                  <li><a href="/rootSet">RootSet</a></li>
                  <li class="divider"></li>
                  <li class="nav-header">Instance Counts</li>
                  <li><a href="/showInstanceCountsIncPlatform/">Including platform</a></li>
                  <li><a href="/showInstanceCountsExcPlatform/">Excluding platform</a></li>
                  <li class="divider"></li>
                  <li><a href="/histogram/">Heap histogram</a></li>
                  <li class="divider"></li>
                  <li><a href="/finalizerSummary/">Finalizer summary</a></li>
                </ul>
              </li>
              <li><a href="/about">About</a></li>
            </ul>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container">

<h1>instance of java.net.SocksSocketImpl #10597</h1>
<table class="table table-bordered table-condensed">
<tr>
<td>
<b>Instance of:</b>
</td>
<td>
<a href="/object/0x7fb29c118">class java.net.SocksSocketImpl</a>
</td>
</tr>
<tr>
<td>
<b>Size</b>
</td>
<td>
120 bytes
</td>
</tr>
<tr>
<td>
<b>Retained Size</b>
</td>
<td>
152 bytes
</td>
</tr>
<tr>
<td>
<b>References to this object</b>
</td>
<td>
<a href="#refers">2</a>
</td>
</tr>
<tr>
<td>
<b>Root References</b>
</td>
<td>
<a href="#rootRefs">1</a>
</td>
</tr>
</table>
<h2>Instance Fields:</h2>
<table class="table table-bordered table-condensed">
<tr>
<th>
Field
</th>
<th>
Type
</th>
<th>
Value
</th>
</tr>
<tr>
<td>
java.net.PlainSocketImpl.CONNECTION_NOT_RESET
</td>
<td>
int
</td>
<td>
0</td>
</tr>
<tr>
<td>
java.net.PlainSocketImpl.CONNECTION_RESET
</td>
<td>
int
</td>
<td>
2</td>
</tr>
<tr>
<td>
java.net.PlainSocketImpl.CONNECTION_RESET_PENDING
</td>
<td>
int
</td>
<td>
1</td>
</tr>
<tr>
<td>
java.net.PlainSocketImpl.anyLocalBoundAddr
</td>
<td>
object
</td>
<td>
<a href="/object/0x7f4555a30">java.net.Inet4Address #1 (32 bytes)</a>
</td>
</tr>
<tr>
<td>
java.net.PlainSocketImpl.closePending
</td>
<td>
boolean
</td>
<td>
false</td>
</tr>
<tr>
<td>
java.net.PlainSocketImpl.fd1
</td>
<td>
object
</td>
<td>
&lt;null&gt;</td>
</tr>
<tr>
<td>
java.net.PlainSocketImpl.fdLock
</td>
<td>
object
</td>
<td>
<a href="/object/0x7f4554e30">java.lang.Object #40 (16 bytes)</a>
</td>
</tr>
<tr>
<td>
java.net.PlainSocketImpl.fdUseCount
</td>
<td>
int
</td>
<td>
1</td>
</tr>
<tr>
<td>
java.net.PlainSocketImpl.lastfd
</td>
<td>
int
</td>
<td>
-1</td>
</tr>
<tr>
<td>
java.net.PlainSocketImpl.resetLock
</td>
<td>
object
</td>
<td>
<a href="/object/0x7f4554e48">java.lang.Object #41 (16 bytes)</a>
</td>
</tr>
<tr>
<td>
java.net.PlainSocketImpl.resetState
</td>
<td>
int
</td>
<td>
0</td>
</tr>
<tr>
<td>
java.net.PlainSocketImpl.shut_rd
</td>
<td>
boolean
</td>
<td>
false</td>
</tr>
<tr>
<td>
java.net.PlainSocketImpl.shut_wr
</td>
<td>
boolean
</td>
<td>
false</td>
</tr>
<tr>
<td>
java.net.PlainSocketImpl.socketInputStream
</td>
<td>
object
</td>
<td>
&lt;null&gt;</td>
</tr>
<tr>
<td>
java.net.PlainSocketImpl.stream
</td>
<td>
boolean
</td>
<td>
true</td>
</tr>
<tr>
<td>
java.net.PlainSocketImpl.timeout
</td>
<td>
int
</td>
<td>
0</td>
</tr>
<tr>
<td>
java.net.PlainSocketImpl.trafficClass
</td>
<td>
int
</td>
<td>
0</td>
</tr>
<tr>
<td>
java.net.SocketImpl.address
</td>
<td>
object
</td>
<td>
<a href="/object/0x7f4555a30">java.net.Inet4Address #1 (32 bytes)</a>
</td>
</tr>
<tr>
<td>
java.net.SocketImpl.fd
</td>
<td>
object
</td>
<td>
<a href="/object/0x7f4555a60">java.io.FileDescriptor #9 (28 bytes)</a>
</td>
</tr>
<tr>
<td>
java.net.SocketImpl.localport
</td>
<td>
int
</td>
<td>
7532</td>
</tr>
<tr>
<td>
java.net.SocketImpl.port
</td>
<td>
int
</td>
<td>
0</td>
</tr>
<tr>
<td>
java.net.SocketImpl.serverSocket
</td>
<td>
object
</td>
<td>
<a href="/object/0x7f4554a00">java.net.ServerSocket #1 (36 bytes)</a>
</td>
</tr>
<tr>
<td>
java.net.SocketImpl.socket
</td>
<td>
object
</td>
<td>
&lt;null&gt;</td>
</tr>
<tr>
<td>
java.net.SocksSocketImpl.applicationSetProxy
</td>
<td>
boolean
</td>
<td>
false</td>
</tr>
<tr>
<td>
java.net.SocksSocketImpl.cmdIn
</td>
<td>
object
</td>
<td>
&lt;null&gt;</td>
</tr>
<tr>
<td>
java.net.SocksSocketImpl.cmdOut
</td>
<td>
object
</td>
<td>
&lt;null&gt;</td>
</tr>
<tr>
<td>
java.net.SocksSocketImpl.cmdsock
</td>
<td>
object
</td>
<td>
&lt;null&gt;</td>
</tr>
<tr>
<td>
java.net.SocksSocketImpl.external_address
</td>
<td>
object
</td>
<td>
&lt;null&gt;</td>
</tr>
<tr>
<td>
java.net.SocksSocketImpl.port
</td>
<td>
int
</td>
<td>
1080</td>
</tr>
<tr>
<td>
java.net.SocksSocketImpl.server
</td>
<td>
object
</td>
<td>
&lt;null&gt;</td>
</tr>
<tr>
<td>
java.net.SocksSocketImpl.useV4
</td>
<td>
boolean
</td>
<td>
false</td>
</tr>
</table>
<a name="rootRefs"></a>
<h2>Root References:</h2>
<ul>
<li>
Java Local Reference from java.lang.Thread #8
</li>
</ul>
<h2>References to this object:</h2>
<a name="refers"></a>
<a href="/object/0x7f4554e08">java.lang.ref.Finalizer #189 (64 bytes)</a>
 : field referent (3/5)
<br/>
<a href="/object/0x7f4554a00">java.net.ServerSocket #1 (36 bytes)</a>
 : field impl (1/2)
<br/>
<h2>Other Queries</h2>
Reference Chains from Rootset
<ul>
<li>
<a href="/objectRootsExcWeak/0x7f4554d90">Exclude weak refs</a>
</li>
<li>
<a href="/objectRootsIncWeak/0x7f4554d90">Include weak refs</a>
</li>
</ul>
<br/>
    </div> <!-- /container -->
    <script src="/js/jquery-2.0.3.min.js"></script>
    <script src="/js/bootstrap.js"></script>
  </body>
</html>

